assets/locales/debugger.properties

```diff
@@ -273,9 +273,9 @@ sourceTabs.closeTab=Close tab
 # for closing the other tabs.
 sourceTabs.closeOtherTabs=Close others
 
-# LOCALIZATION NOTE (sourceTabs.closeTabsToRight): Editor source tab context menu item
-# for closing the tabs to the right of the selected tab.
-sourceTabs.closeTabsToRight=Close tabs to the right
+# LOCALIZATION NOTE (sourceTabs.closeTabsToEnd): Editor source tab context menu item
+# for closing the tabs to the end of the selected tab.
+sourceTabs.closeTabsToEnd=Close tabs to the end
```

Comment:
Hmm, @clarkbw should this be `Close tabs to the end` or `Close tabs to the right`?

My understanding is that it can still be "right" and translators will know how to fix this with RTL languages